packages/ruby/spec/readme/har_request_spec.rb

```diff
@@ -66,10 +66,10 @@
         body: {key1: "key1", key2: "key2"}.to_json
       )
       filter_params = ["Filtered-Header", "key1"]
-      request = Readme::HarRequest.new(http_request, filter_params)
+      request = Readme::HarRequest.new(http_request, Filter.for(filter_params))
       json = request.as_json
 
-      request_body = json.dig(:postData, :text)
+      request_body = JSON.parse(json.dig(:postData, :text))
```

Comment:
Hmmm is this asserting that the body is doubly encoded?

Reply:
I'm not sure about this. When I introduced the `HarCollection` object, I was unable to send data to the Metrics API with our sample application. Adding the `to_json` call on the request body fix that issue, and so I updated this test accordingly. It seems like introducing the `HarCollection` changed something, but its not clear what.

Reply:
I ran into this exact same issue on my branch. Even the schema was complaining. Not sure how it was not failing spec before 🤷